@here/harp-geoutils/lib/projection/Projection.ts

```diff
@@ -89,8 +95,10 @@ export interface Projection {
      * ```
      *
      * @param worldBox The bounding box in world coordinates.
+     * @param clamp Whether to clamp the world points, if not clamped, the
```

Comment:
I would like to suggest to indent the lines in the @param section:

@param clamp Whether to clamp the world points, if not clamped, the .... 100 chars are not enough....
-> -> indent by two tabs

This seems to make the code more readable. Just a _suggestion_...

Reply:
I will remove this parameter, but will take your suggestion into consideration in future.